### DISCUSSION TITLE
R4phd - RRR – Structuring your project

### DISCUSSION THREAD

**Discussion by giscus** [sorenoneill/r4phd#14] [General]
# R4phd - RRR – Structuring your project

https://sorenoneill.github.io/r4phd/qmd_rrr/reproducible_research_workflow_2_en.html

**Comment by jenshk77** [comment]
Hej Søren
tak for de fine øvelser. Det er nogle quarto-filer man gerne vil gemme til inspiration senere.
Men hvad er reelt forskellen på static og dynamic presentation?
Og hvor skal man begynde, hvis man gerne vil lege med geografisk data i R?

**Reply by sorenoneill** [reply]
Der er vist ingen forskel på statisk og dynamisk ... Jeg har ikke fået skrevet den dynamiske endnu ... Det står vist derinde et eller andet sted

Ift geografiske data kan du starte her https://www.bookdown.org/mcwimberly/gdswr-book/
